webapp/channels/src/components/edit_post/edit_post.tsx

```diff
@@ -157,6 +161,35 @@ const EditPost = ({editingPost, actions, canEditPost, config, channelId, draft,
     // just a helper so it's not always needed to update with setting both properties to the same value
     const setCaretPosition = (position: number) => setSelectionRange({start: position, end: position});
 
+    useEffect(() => {
+        function onPaste(event: ClipboardEvent) {
```

Comment:
We already have a `pasteHandler` in `webapp/channels/src/utils/paste.tsx`.

Is this something we can reuse in any way? or are there strong reasons to keep this one different? (I see it is for the most part the same, but missing some things).

Reply:
there's no major differences besides the fact that uses some props that don't exist in `webapp/channels/src/components/edit_post/edit_post.tsx`.  An alternative method is to modify `pasteHandler()` to replace this logic which I originally didn't want to do but it does make sense for the logic for the post textbox and the edit textbox to be the same.

How would you want to approach this?

Reply:
Let's go with updating the behavior in `pasteHandler`. Without having looked at it much, what I would do is pass a special location from this component (maybe add it to the Locations constants as "EDIT"), and handle the differences based on that.